src/jobs/deploy-eb.yml

```
@@ -1,9 +1,88 @@
 description: >
-  This job deploys an application to aws elasticbeanstalk.
+  This job deploys an application to aws elasticbeanstalk. Deploy-Eb requires publish-docker
 
 executor: node/default
 
+parameters:
+  # Required Parameters
+  application-name:
+    type: string
+    description: |
+      The name of the elastic beanstalk instance that will be updated in this job
+  environment-name:
+    type: string
+    description: |
+      The name of the elastic beanstalk instance that was given during eb init
+  repo-name:
+    type: string
+    description: |
+      The Github repository name of the elasticbeanstalk instance
+      This will be used for tagging
+  version:
+    type: string
+    description: |
+      The semantic version of the eb instance
+  # Not Required Parameters
+  aws-secret-access-key-id:
+    type: env_var_type
+    default: AWS_SECRET_ACCESS_KEY_ID
```

in our org context the secret key for EB is MACHINE_AWS_EB_SecretKey but i had a convo with chris who said we are changing them to all uppercase
```suggestion
    default: MACHINE_AWS_EB_SECRETKEY
```